airbyte-integrations/connectors/source-clazar/metadata.yaml

```
@@ -16,7 +16,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: d7df7b64-6266-45b5-ad83-e1515578f371
-  dockerImageTag: 0.3.0
+  dockerImageTag: 1.0.0
```

```suggestion
  dockerImageTag: 0.4.0
```

Your PR does not have a breaking change documentation. You would have to either add the breaking change docs, or call this a minor version bump and then I can merge this as is.